src/config/browser-options.js

```diff
@@ -189,14 +189,19 @@ function buildBrowserOptions(defaultFactory, extra) {
                     throw new Error('"screenshotsDir" must be a string or function');
                 }
             },
-            map: (value, _, __, { isSetByUser }) => {
-                const deprecatedScreensPath = "hermione/screens";
+            map: (value, config, __, { isSetByUser }) => {
+                if (isSetByUser) {
+                    return value;
+                }
 
-                if (!isSetByUser && fs.existsSync(deprecatedScreensPath) && !fs.existsSync(value)) {
-                    return deprecatedScreensPath;
+                const topLevelScreenshotsDir = _.get(config, "screenshotsDir");
```

Comment:
for top level option this will be `undefined` because option is not set. But when parse browser this option will have value from top level. And will be used if user not set it specially for browser